src/app/records/PresentationNode.tsx

```diff
@@ -81,6 +83,17 @@ export default function PresentationNode({
 
   const title = <PresentationNodeTitle def={nodeDef} overrideName={overrideName} />;
 
+  // makes shallow copy so original node will be unsorted, makes toggle work
+  const sortedNode = { ...node };
+  if (sortRecordProgression && sortedNode.acquired) {
+    sortedNode.childPresentationNodes = orderBy(
+      sortedNode.childPresentationNodes,
+      (entry) => entry['acquired'] / entry['visible'],
```

Comment:
You shouldn't need to use string properties here:
```suggestion
      (entry) => entry.acquired / entry.visible,
```

Reply:
The reason why had it like that was because I was originally getting a `Property does not exist on type 'never'` error while working on it. I changed and it does not seem to be having this error anymore.